packages/core/src/messagingProtocols/mls/index.ts

```diff
@@ -18,3 +18,4 @@
  */
 
 export * from './MLSService/';
+export * from './types';
```

Comment:
So we can import it 
`import {...} from '@wireapp/core/messagingProtocols/mls'`
instead of 
`import {...} from '@wireapp/core/messagingProtocols/mls/types'`